config/checkstyle/suppressions.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+        "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+        "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+    <suppress files=".*\.java" checks="HideUtilityClassConstructorCheck"/>
```

Comment:
Personally, I'm accustomed to adding a private constructor to mitigate the risks (small though they may be) of allowing construction of utility classes.

Reply:
I'm not sure if this is a reason why it's been added as a suppression, but when I removed this and ran checkstyle locally, it flags the entry class `IntegrationAdapterLabResultsApplication.java` as a utility class

Reply:
Yeah because it doesn't have any non-static methods (except those inherited from `Object`). In most cases though, we should be interested to know something has manifested as a utility class as it might not have been intentional. I'd prefer us to remove this suppression and call out the Application class specifically as a FP.

Reply:
Done in 1774eef